src/drivers/dw/dma.c

```diff
@@ -1234,10 +1190,12 @@ static void dw_dma_irq_handler(void *data)
 		return;
 	}
 
+#if DW_USE_HW_LLI
```

Comment:
strictly speaking adding these "#if's" is unrelated... Would be good to have a separate patch with a proper description for that

Reply:
It's related in the manner, that after removing dw_dma_work function some platforms won't build.